homeassistant/components/madvr/strings.json

```diff
@@ -38,23 +38,20 @@
       }
     },
     "sensor": {
-      "mac_address": {
-        "name": "Mac address"
-      },
       "temp_gpu": {
-        "name": "Gpu temperature"
+        "name": "Temp gpu"
       },
       "temp_hdmi": {
-        "name": "Hdmi temperature"
+        "name": "Temp hdmi"
       },
       "temp_cpu": {
-        "name": "Cpu temperature"
+        "name": "Temp cpu"
       },
       "temp_mainboard": {
-        "name": "Mainboard temperature"
+        "name": "Temp mainboard"
```

Comment:
I liked the old name better since it was more readable. `GPU temperature` looks better imo